pro/.storybook/preview-head.html

```diff
@@ -1,3 +1,8 @@
 <style>
-  @import 'design-system/build/css/font-faces.css';
+  @import '../src/styles/index.scss';
```

Comment:
C'est pas un truc du genre qu'il faut faire ? (en enlevant la balise style)
```suggestion
    <link rel="stylesheet" type="text/css" href="index.css">
```